astroquery/mast/tests/test_mast.py

```diff
@@ -49,6 +52,8 @@ def post_mockreturn(method="POST", url=None, data=None, timeout=10, **kwargs):
     else:
         service = re.search(r"service%22%3A%20%22([\w\.]*)%22", data).group(1)
 
+    print(service)
```

Comment:
I'm fairly certain that this is not needed here

Reply:
Yupppp! Good catch, removed. :)